ansible/roles/kong-api/defaults/main.yml

```diff
@@ -5665,3 +5665,124 @@ kong_apis:
       - name: request-size-limiting
         config.allowed_payload_size: "{{ medium_request_size_limit }}"
 
+  - name: updateUserV2
+      uris: "{{ user_service_prefix }}/v2/update"
+      upstream_url: "{{ learning_service_url }}/v2/user/update"
+      strip_uri: true
+      plugins:
+        - name: jwt
+        - name: cors
+        - "{{ statsd_pulgin }}"
+        - name: acl
+          config.whitelist:
+            - 'userUpdate'
+        - name: rate-limiting
+          config.policy: local
+          config.hour: "{{ medium_rate_limit_per_hour }}"
+          config.limit_by: credential
+        - name: request-size-limiting
+          config.allowed_payload_size: "{{ medium_request_size_limit }}"
+
+  - name: managedUserV1Create
+      uris: "{{ user_service_prefix }}/v1/managed/create"
+      upstream_url: "{{ learning_service_url }}/v1/manageduser/create"
+      strip_uri: true
+      plugins:
+        - name: jwt
+        - name: cors
+        - "{{ statsd_pulgin }}"
+        - name: acl
+          config.whitelist:
+            - 'userAccess'
+        - name: rate-limiting
+          config.policy: local
+          config.hour: "{{ medium_rate_limit_per_hour }}"
+          config.limit_by: credential
+        - name: request-size-limiting
+          config.allowed_payload_size: "{{ medium_request_size_limit }}"
+
+  - name: ssoUserV1Create
```

Comment:
is this used by mobile devices?

Reply:
only in portal

Reply:
Hi @SMY-ALTAMASH  The documentation pr is ready hari will be merging that after he have a look in to it.